lib/application.js

```diff
@@ -295,7 +295,8 @@ app.enableAuth = function() {
   var remotes = this.remotes();
   var app = this;
 
-  remotes.before('**', function(ctx, next, method) {
+  remotes.authorization = function(ctx, next) {
```

Comment:
Should we save the original `authorization` implementation and call after loopback auth passed? I think that's the canonical way of overriding methods like this.

```js
var baseAuthorization = remotes.authorization;
remotes.authorization = function(ctx, next) {
  if (authorized) {
    baseAuthorization(ctx, next); // or perhaps .call(this, ctx, next)
  } else {
    next(new Error('Not authorized.'));
  }
};
```